packages/shared/useTimeoutFn/index.ts

```diff
@@ -27,15 +28,19 @@ export function useTimeoutFn(
 
   function stop() {
     isActive.value = false
+    isPending.value = false
     clear()
   }
 
-  function start() {
+  function start(...args: unknown[]) {
     clear()
     isActive.value = true
+    isPending.value = true
     timer = setTimeout(() => {
+      isPending.value = false
       timer = null
-      cb()
+      // eslint-disable-next-line
```

Comment:
Can you make this more specific by only disabling the rule that throws?

Reply:
okay